Unicode/is_Ligature.c

```diff
@@ -1,104 +1,21 @@
 /*
 Copyright: 2012 Barry Schwartz
+    Create is_Ligature.c (using a python script) to test for Vulgar fractions
 Copyright: 2016 Joe Da Silva
-License: BSD-3-clause
+    Re-write is_Ligature.c to test for ligatures, Vulgar and Other-fractions.
+Copyright: 2016 Gioele Barabucci
+    Simplify makeutype.c code. Split is_Ligature.c, create is_Ligature_data.h
+License: GPL-3+
```

Comment:
I thought it was project policy to use GPL3+ on new code, and only preserve existing licenses on old code.  So if this really _is_ a re-write and not just a modification of what was there, GPL3+ would be appropriate.